core/network/impl/protocols/parachain_protocol.hpp

```diff
@@ -54,11 +54,12 @@ namespace kagome::network {
                       const primitives::BlockHash &genesis_hash,
                       std::shared_ptr<ObserverType> observer,
                       Protocol const &protocol,
-                      std::shared_ptr<network::PeerView> peer_view)
+                      std::shared_ptr<network::PeerView> peer_view,
+                      log::Logger logger)
         : base_(kParachainProtocolName,
                 host,
-                make_protocols(protocol, genesis_hash, "polkadot"),
-                log::createLogger("ParachainProtocol", protocol)),
+                make_protocols(protocol, genesis_hash, chain_spec),
```

Comment:
[rust](https://github.com/paritytech/polkadot/blob/3fd990501423edca102bc473c2bb1c2ce9b93c23/node/network/protocol/src/peer_set.rs#L32-L33)
```suggestion
                make_protocols(protocol, genesis_hash, "polkadot"),
```

Reply:
as above